Civi/Funding/PayoutProcess/Validator/DrawdownReviewValidator.php

```diff
@@ -79,7 +82,9 @@ public function validate(AbstractEntity $new, AbstractEntity $current): EntityVa
   public function validateNew(AbstractEntity $new): EntityValidationResult {
     if ('new' !== $new->getStatus()) {
       $fundingCase = $this->getFundingCase($new);
-      $this->assertPermission($fundingCase);
+      if (!$this->hasReviewPermission($fundingCase->getPermissions())) {
+        $this->assertPermission($fundingCase);
+      }
```

Comment:
why does `$fundingCase` needs a _'review_drawdown'_ permission if no review permissions are detected by `hasReviewPermission()`?

would it make sense to assert that _'review_drawdown'_ is necessary if `hasReviewPermission()` returns true?

Reply:
It allows clearing reviewers to create a final drawdown when finishing clearing. I hope the minor change in this file makes this more clear.